### ISSUE TITLE
Is there a way to use file extension instead of mime types to filter what will be minified?

### ISSUE THREAD

**Issue by EthraZa** [tdewolff/minify#315]
These are my files and, as you can see, the mime types does not match what minify believes a css is, for example.

```
# file -i in/*
in/a.css:  text/plain; charset=us-ascii
in/a.html: text/html; charset=us-ascii
in/a.js:   text/plain; charset=utf-8
in/a.php:  text/x-php; charset=utf-8
in/a.xml:  application/xml; charset=us-ascii
```

What I want is to sync all but minify only the html, css, js and json files. The php, xml, xss, txt, log, zip needs to be only synced as is.

Please advise.
Thank you.

**Comment by tdewolff** [comment]
Thanks for the report, it seems some functionality is not working so well. First of all, the mimetype of files is deduced from the file extension. That is, `a.css` is treated as having a `text/css` mimetype so there should be no problem there. In order to minify HTML, CSS, JS, and JSON files and to sync all other files (including XML), the following command should achieve that but unfortunately doesn't:

```
minify --match "\.[css|html|js]$" --sync -o out *
```

I will work on a solution.

**Comment by tdewolff** [comment]
Fixed in 01cd981, let me bring out a new release

**Comment by tdewolff** [comment]
Please verify if release v2.7.7 works for you, otherwise reopen the issue. Thanks!
